src/ig/files/_updatePublisher.sh

```diff
@@ -1,9 +1,48 @@
 #!/bin/bash
+dlurl=https://fhir.github.io/latest-ig-publisher/org.hl7.fhir.publisher.jar
+publisher_jar=org.hl7.fhir.publisher.jar
+input_cache_path=./input-cache/
+
 set -e
 if ! type "curl" > /dev/null; then
-  echo "ERROR: Script needs curl to download latest IG Publisher. Please install curl."
-  exit 1
+	echo "ERROR: Script needs curl to download latest IG Publisher. Please install curl."
+	exit 1
 fi
-echo "Downloading most recent publisher - it's ~100 MB, so this may take a bit"
-curl https://fhir.github.io/latest-ig-publisher/org.hl7.fhir.publisher.jar -o input-cache/org.hl7.fhir.publisher.jar --create-dirs
-echo "Done"
\ No newline at end of file
+
+publisher="$input_cache_path$publisher_jar"
+if test -f "$publisher"; then
+	echo "IG Publisher FOUND in input-cache"
+	jarlocation="$publisher"
+	jarlocationname="Input Cache"
+	upgrade=true
+else
+	publisher="../$publisher_jar"
+	upgrade=true
+	if test -f "$publisher"; then
+		echo "IG Publisher FOUND in parent folder"
+		jarlocation="$publisher"
+		jarlocationname="Parent Folder"
+		upgrade=true
+	else
+		echo IG Publisher NOT FOUND in input-cache or parent folder...
+		jarlocation=$input_cache_path$publisher_jar
+		jarlocationname="Input Cache"
+		upgrade=false
+	fi
+fi
+
+if "$upgrade"; then
+	message="Overwrite $jarlocation?"
+else
+	echo Will place publisher jar here: "$jarlocation"
+	message="Ok?"
+fi
+
+read -r -p "$message" response
```

Comment:
Personally, I would like a space between the question and the input (rather than my typed response being the character immediately following the question -- currently `OK?Y`). How do you feel about putting a space in (e.g., `"$message "`)?

Does Windows have the same behaviour or does batch insert a space for you?

Reply:
Same behavior on windows, you get `(Y/N)Y`

Reply:
@ngfreiter did you notice this missing space? I can easily add it in if it's missing.